modules/mobile/lib/mobile/engine.rb

```diff
@@ -7,6 +7,9 @@ class Engine < ::Rails::Engine
     isolate_namespace Mobile
```

Comment:
This Engine class is being loaded via `modules/mobile/bin`, which is how this is being used despite also being in the lib directory.